src/main/java/io/qameta/allure/bamboo/AllureExecutable.java

```diff
@@ -26,7 +26,7 @@
     AllureGenerateResult generate(Path sourceDir, Path targetDir) {
         try {
             forceMkdir(targetDir.toFile());
-            final LinkedList<String> args = new LinkedList<>(asList("generate", "-o", targetDir.toString(), "-v", sourceDir.toString()));
+            final LinkedList<String> args = new LinkedList<>(asList("generate", "-c", "-o", targetDir.toString(), sourceDir.toString()));
```

Comment:
@baev provided directory might exist (in fact it does, because we're using temporary dir). I tested it with 2.0 and it fails if directory exists (even if it's empty). 
This will not work with BETA5. I don't think it should though.